spring-cloud-stream-binder-kafka-docs/src/main/asciidoc/overview.adoc

```diff
@@ -430,11 +430,13 @@ On the other hand, if auto topic creation is disabled on the server, then care m
 
 If you want to have full control over how partitions are allocated, then leave the default settings as they are, i.e. do not exclude the kafka broker jar and ensure that `spring.cloud.stream.kafka.binder.autoCreateTopics` is set to `true`, which is the default.
 
-== Kafka Stream Binding Capabilities of Spring Cloud Stream
+== Kafka Streams Binding Capabilities of Spring Cloud Stream
 
 Spring Cloud Stream Kafka support also includes a binder specifically designed for Kafka Streams binding.
 Using this binder, applications can be written that leverage the Kafka Streams API.
-For more information on Kafka Streams, see https://kafka.apache.org/0110/documentation/streams/developer-guide[Kafka Streams API Developer Manual]
+For more information on Kafka Streams, see https://kafka.apache.org/documentation/streams/developer-guide[Kafka Streams API Developer Manual]
+
+Kafka Streams support in Spring Cloud Stream is based on the foundations provided by the Spring Kafka project. For details on that support, see http://docs.spring.io/spring-kafka/docs/1.1.6.RELEASE/reference/html/_reference.html#kafka-streams[Kafaka Streams Support in Spring Kafka].
```

Comment:
The rule about links: avoid versions as much as possible.
There is http://docs.spring.io/spring-kafka/reference/html/_reference.html#kafka-streams alias pointing to the latest version of the project. This way, when you upgrade and go ahead with your new version, there won't be need to take a look into the docs to fix old versions mentionings